### PR TITLE
Update datashader docs

### PR DESCRIPTION
**Pull request by asinghvi17**
We don't necessarily need to specify the axis anymore, since the default axis for all plots is `Axis`.

Should we also consider hiding the `async=false` from the docs?  If so then users can copy paste immediately and get full performance.  Otherwise, using this interactively can segfault because of the number of events...

### CODE REVIEW

**Comment by SimonDanisch** [comment]
Actually I'd rather default to `async=false` since I noticed that it segfaults from time to time 😅

**Comment by asinghvi17** [comment]
I got segfaults with async=false...

**Comment by asinghvi17** [comment]
I guess these changes can be merged now though, and I'll try to reproduce the segfault at some point...

File: docs/src/reference/plots/datashader.md [hunk 1]
```diff
@@ -15,13 +15,10 @@ using DelimitedFiles
 airports = Point2f.(eachrow(readdlm(assetpath("airportlocations.csv"))))
 fig, ax, ds = datashader(airports,
     colormap=[:white, :black],
-    # use type=Axis, so that Makie doesn't need to infer
-    # the axis type, which can be expensive for a large amount of points
-    axis = (; type=Axis),
     # for documentation output we shouldn't calculate the image async,
```
Comment (SimonDanisch) [inline docs/src/reference/plots/datashader.md:18]: Why would we remove this?

**Comment by SimonDanisch** [comment]
Thanks!

**Pull request merged**